var/spack/repos/builtin/packages/qemu/package.py

```diff
@@ -0,0 +1,180 @@
+    version('1.2.1',      sha256='7ccee3c0d67cf506d85ab1bb80b8bc6734fff7f610c34745798e3945d68a53fc')
+    version('1.2.0',      sha256='ba1fc4cc582d3e6f267a72d864c4c8d2f7ee53b160ef338ae4e64e041e6b6c11')
+    version('1.1.2',      sha256='715aba5e05d48a3af69f704840e8a478a0474d0fb75d846e197dac8942c29ad4')
+    version('1.1.1',      sha256='e28722fd676fed34cfdc07581fd2547a766a1785c71f391ed1f2f4f4e923df64')
+    version('1.1.0-1',    sha256='3a58fbaef1715ac3fa40e534d499873b66039b6adb409a22df11d45a5332e21c')
+    version('1.1.0',      sha256='d9899cf9842f2fe4b0dc3c659dabc8ae32e192151f5be3709a1cd7be5a2c0a9a')
+    version('1.0.1',      sha256='a0548ac6221fce09a5a6877c8db19e8755e175f97b934e69a449170a604825dd')
+    version('1.0',        sha256='6b75305f49cdff890df52a3a341060e8909592d8a9a9b330bb6808153b0a2078')
+    version('0.15.1',     sha256='e25d068d753d1a7d498ae6fdf916da08c3a1aa9ddfb4c5078750009359bb3a75')
+    version('0.15.0',     sha256='6a319f56b9f68c9de3f0b4bf5a6a47e065f4bde50e64f715e12819b2e6f92e4d')
+    version('0.14.1',     sha256='274de48593d153d4cb4c487c902eafcdbd3224f1b846ff832aba03396d8453c7')
+    version('0.14.0',     sha256='fbe4e1d04bf57faeb53c6ae2a2a280504a3c8c9cd1f47db6180c1f8d7e48364d')
+    version('0.13.0',     sha256='69ff5503313fbfcb7b911f2129f16f61a61c7d1a7f13e73f8d6537eaac31e924')
+    version('0.12.4',     sha256='935b243bcf461bb0c6e6c938cd21498edbe204861c4db0869a42cca2f33ff95c')
+    version('0.12.3',     sha256='813cbbe26d521f1081774a4e041b3f068f41481071ba01a4154b4876eaafea90')
+    version('0.12.2',     sha256='26a67f6a9e288117833e498cc02ed61e1e9439e6275ddfe2478da62cdf20e9ba')
+    version('0.12.1',     sha256='2865bde4ab2c53e257755cb8f281ee6fba5a9bf54bc3c503edfa36dc03b484fd')
+    version('0.12.0',     sha256='e885a7bdd8923f828aba3a924ad992c81c0c67f80265279017dd71b5acafd84e')
+    version('0.11.0',     sha256='cc1d141a91e180c9a151369592bcd6b864c9bf581f2db837ab365aec1533bb3e')
+    version('0.10.6',     sha256='5727770bb8cea75e6bd76e0911008217a0fd07ed84ae82d967c7bdd98f9c7749')
+    version('0.10.5',     sha256='59ac82c32b6493bf0a6042ef39549368ad8f0d3ec2151ab040f4ce28c93d751e')
+    version('0.10.4',     sha256='db46889bc8908318e15f26f4bde8eb31824cb117d9234afb77a348ad4b6f5050')
+    version('0.10.3',     sha256='4a5393ff47493a80d28a53ce307bffa52a066fd82045a5598b8da074a2f7dfe7')
+    version('0.10.2',     sha256='2d62bcad5ef7609813381e77a6422fec3d4cbd6347f4297b687510ab4d32e813')
+    version('0.10.1',     sha256='632b8942d8c85b36997ce3ed893bc34c868b432fbaadd4ea86994ca881d6665b')
+    version('0.10.0',     sha256='cd554729fa9d0ec17164afbc1cea62d02bde3db8e16db3fd1b8e71d8e1b3dd41')
+    version('0.9.1',      sha256='a9655a471d0649f5540b890447b35849c162d9b986bf2bbddcb68461748e0f42')
+
+    # FIXME: Add dependencies if required.
+    # glib-2.40 gthread-2.0 is required to compile QEMU
+    depends_on('glib@2.40:')
+    depends_on('pixman') # software reports requirement >= 0.21.8
```

Comment:
```suggestion
    depends_on('pixman@0.21.8:')
```
should work fine, if not there's a bug.